pages/projects/[projectId]/collections/[collectionId]/traits/[traitId]/values/import-list.tsx

```diff
@@ -0,0 +1,261 @@
+                trait?.name +
+                "' trait. Provide us a csv-formatted file where each line looks like: `VALUE_NAME,RARITY`, where `RARITY` is decimal (.1 == 10%). The sum of all RARITY values should be 1."
+              }
+            />
+            <div className="mt-5 md:mt-0 md:col-span-2">
+              <form action="#" method="POST" onSubmit={onSubmit}>
+                <div className="shadow sm:rounded-md sm:overflow-hidden">
+                  <div className="px-4 py-5 bg-white space-y-6 sm:p-6">
+                    <div>
+                      <label
+                        htmlFor="name"
+                        className="block text-sm font-medium text-gray-700"
+                      >
+                        Upload File
+                      </label>
+                      <label
+                        htmlFor="files"
+                        className="cursor-pointer w-full mt-1 flex justify-center px-6 pt-20 pb-20 border-2 border-gray-300 border-dashed rounded-md"
+                      >
+                        <div className="space-y-1 text-center">
+                          <svg
+                            className="mx-auto h-18 w-12 text-gray-400"
+                            stroke="currentColor"
+                            fill="none"
+                            viewBox="0 0 48 48"
+                            aria-hidden="true"
+                          >
+                            <path
+                              d="M28 8H12a4 4 0 00-4 4v20m32-12v8m0 0v8a4 4 0 01-4 4H12a4 4 0 01-4-4v-4m32-4l-3.172-3.172a4 4 0 00-5.656 0L28 28M8 32l9.172-9.172a4 4 0 015.656 0L28 28m0 0l4 4m4-24h8m-4-4v8m-12 4h.02"
+                              strokeWidth={2}
+                              strokeLinecap="round"
+                              strokeLinejoin="round"
+                            />
+                          </svg>
+                          <span className="relative bg-white rounded-md text-sm font-medium text-indigo-600 hover:text-indigo-500 focus-within:outline-none focus-within:ring-2 focus-within:ring-offset-2 focus-within:ring-indigo-500">
+                            Upload CSV file
+                          </span>
+                          <input
+                            id="files"
+                            name="files"
+                            type="file"
+                            className="sr-only"
+                            accept=".csv"
+                            onChange={(e) => setSelectedFiles(e.target.files)}
+                          />
+                          <p className="text-xs text-gray-500 pb-5">
+                            Please select a file with comma-separated values.
+                          </p>
+                          <p className="text-xs text-gray-500 pb-5">
+                            {/* {selectedFiles
+                              ? "[" + selectedFiles?.length + " files selected]"
+                              : ""} */}
```

Comment:
i think it makes sense to uncomment this section too - it helps to show that something happened after selecting a file

Reply:
done!